Add doc to switch between different JDK versions

shireeshaBongarala opened the pull request:
https://github.com/AdoptOpenJDK/homebrew-openjdk/issues/111

aahlenst commented:
In my experience, `export JAVA_HOME=$(/usr/libexec/java_home -v 13)` is sufficient. Alterations of `$PATH` rather hurt than help on macOS. `/usr/bin/java` is the default `java` on macOS and provides the magic that makes `JAVA_HOME` work. If you prepend the output of `export JAVA_HOME=$(/usr/libexec/java_home -v 13)` to `$PATH`, you break all the magic.

shireeshaBongarala commented:
@aahlenst Thanks for the comment. I fixed the issue in my local `$PATH` to remove java paths other than `/usr/bin/java`.  Now, just  updating the `JAVA_HOME` env var is sufficient. 
I updated the PR.
